workflows/raredisease.nf

```diff
@@ -147,9 +148,13 @@ workflow RAREDISEASE {
     }
 
     // STEP 1.5: BAM QUALITY CHECK
+    gff = []
+    use_gff = false
```

Comment:
If the plan is for these two parameters to always be empty/false then think they should be defined within the subworkflow rather than in the main file. Else, it needs to be a parameter that one can set and not hardcoded here.